docs/distributions/ubuntu/installation.md

```diff
@@ -1,6 +1,6 @@
 # Download the latest safe release
 
-Warning: Versions newer than 5.7.19 are considered in-development builds and **should not** be used due to issues from within the kernel and the init system.
+Warning: Mac Mini users will need to [install rEFInd](https://wiki.t2linux.org/guides/refind/) as the Mac Startup Manager on Mac Mini is not booting Ubuntu.
```

Comment:
Probably add ", with the default bootloader (GRUB)." to the end of this.
Also can you mention that it is unknown why this is happening.